content/ja/docs/concepts/workloads/pods/podpreset.md

```diff
@@ -0,0 +1,62 @@
+---
+reviewers:
+- jessfraz
```

Comment:
Not sure who reviews localized content. Is it the localization team?

Reply:
It's not the localization team. I'll delete it.